src/components/molecules/LiveSchedule/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./LiveSchedule";
```

Comment:
As above RE: preferencing named exports over using `default`

Reply:
nit/stylistic that we haven't established our stance on yet, resolving this comment